authors/apps/authentication/tests/test_views.py

```diff
@@ -40,3 +40,25 @@ def test_login_with_invalid_user_fails(self):
         self.assertDictEqual(resp.data, expected_response)
         self.assertIn(
             "A user with this email and password was not found.", str(resp.data))
+
+    # test whether a token is generated on login
+    def test_login_user_login_token(self):
+        user=User.objects.create_user(username='sampleuser', email='user@sprinters.ug',password='butt3rfly1')
+        user.is_active=True
+        user.save()
+
+        response = self.client.post(
+            reverse('auth:login'),
+            data=self.user_data,
+            format='json')
+        self.assertIn('token', response.data)
+        self.assertEqual(response.status_code, 200)
+
+# test whether a token is generated on signup
+    def test_signup_user_signup_token(self):
+        response = self.client.post(
+            reverse('auth:register'),
+            data=self.user_data,
+            format='json')
+        self.assertIn('generate_auth_token', response.data)
```

Comment:
For uniformity, you should use the same key identifier for the token for both login and sign up